team.go

```diff
@@ -79,18 +79,31 @@ func (c *Client) Team(id int64) (*Team, error) {
 // AddTeam makes a new team
 // email arg is an optional value.
 // If you don't want to set email, please set "" (empty string).
-func (c *Client) AddTeam(name string, email string) error {
+// When team creation is successful, returns the team id
+func (c *Client) AddTeam(name string, email string) (int64, error) {
+	id := int64(0)
 	path := fmt.Sprintf("/api/teams")
+
 	team := Team{
 		Name:  name,
 		Email: email,
 	}
 	data, err := json.Marshal(team)
 	if err != nil {
-		return err
+		return id, err
 	}
 
-	return c.request("POST", path, nil, bytes.NewBuffer(data), nil)
+	tmp := struct {
+		Id int64 `json:"teamId"`
+	}{}
+
+	err = c.request("POST", path, nil, bytes.NewBuffer(data), &tmp)
+
+	if err != nil {
+		return id, err
+	}
```

Comment:
It's preferable to co-locate the error check with the origin of the error, with readability in mind:
```suggestion
	err = c.request("POST", path, nil, bytes.NewBuffer(data), &tmp)
	if err != nil {
		return id, err
	}
```